src/rect.rs

```diff
@@ -29,6 +29,13 @@ impl<T: fmt::Debug> fmt::Debug for Rect<T> {
     }
 }
 
+impl<T: fmt::Display> fmt::Display for Rect<T> {
+    fn fmt(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+        write!(formatter, "Rect(Size {} at Origin {})", self.size.to_string(), self.origin.to_string());
```

Comment:
Is there a reason why this follows a different format from `Debug`? `Rect({} at {})` seems terser.